test/utils/zombie.ts

```diff
@@ -321,3 +321,52 @@ export async function directoryExists(directoryPath: PathLike) {
         return false;
     }
 }
+
+export async function monitorBlockProduction(apis: ApiPromise[], blockProductionTimeout = 15000): Promise<() => void> {
```

Comment:
Maybe this is too hacky but let's try if it works. Add a comment of what it does. If I understand right:

This will log an error if no blocks are produced in 15 seconds for any chain.

It could cause problems with container 2002 because it starts as not registered, so it may error at the beginning. And also we deregister it later so it may also error at the end of the test.

Reply:
<img width="555" height="191" alt="image" src="https://github.com/user-attachments/assets/c7d453b7-8475-43c2-ae89-137a372e3b4c" />